src/styles/blocks/catalog.scss

```diff
@@ -11,6 +11,7 @@
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
+    transition-duration: 300ms;
```

Comment:
you should set transition to define properties, not to all by default, move duration to variable